We do not have hatchling on Solaris, and it doesn't seem necessary
to deliver it at this point. This patch modifies the build metadata
to use setuptools instead.

Not intended for upstream.

--- jsonschema-4.23.0/pyproject.toml
+++ jsonschema-4.23.0/pyproject.toml
@@ -1,14 +1,19 @@
 [build-system]
-requires = ["hatchling", "hatch-vcs", "hatch-fancy-pypi-readme"]
-build-backend = "hatchling.build"
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
 
 [tool.hatch.version]
 source = "vcs"
 
+[tool.setuptools]
+packages = ["jsonschema"]
+
 [project]
 name = "jsonschema"
+version = "4.23.0"
 description = "An implementation of JSON Schema validation for Python"
 requires-python = ">=3.8"
+readme = "README.rst"
 license = {text = "MIT"}
 keywords = [
     "validation",
@@ -37,7 +42,6 @@ classifiers = [
   "Topic :: File Formats :: JSON",
   "Topic :: File Formats :: JSON :: JSON Schema",
 ]
-dynamic = ["version", "readme"]
 dependencies = [
   "attrs>=22.2.0",
   "jsonschema-specifications>=2023.03.6",
